docker/trustlines/README.md

```diff
@@ -72,16 +74,6 @@ line.
 Of course you can also use an existing keyfile, but please do not
 reuse a keyfile from a validator node.
 
-### Copy compiled contracts
```

Comment:
Please leave this I think, as this is the docker explanation this should be the default I guess. 
You can then mention here that there are other possibilities.

Reply:
:heavy_check_mark: